src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -562,11 +562,11 @@ Either change the value in the AndroidManifest.xml to match the $(SupportedOSPla
 {0} - The assembly name</comment>
   </data>
   <data name="APT2264" xml:space="preserve">
-    <value>This could be caused by the project exceeding the Windows maximum path length limitation. See https://learn.microsoft.com/xamarin/android/errors-and-warnings/apt2264 for details.</value>
+    <value>This could be caused by the project exceeding the Windows maximum path length limitation. See https://learn.microsoft.com/en-us/dotnet/android/messages/apt2264 for details.</value>
     <comment>The following are literal names and should not be translated:</comment>
   </data>
   <data name="APT2265" xml:space="preserve">
-    <value>This could be caused by the project having non-ASCII characters in its filename or path. See https://learn.microsoft.com/xamarin/android/errors-and-warnings/apt2265 for details.</value>
+    <value>This could be caused by the project having non-ASCII characters in its filename or path. See https://learn.microsoft.com/en-us/dotnet/android/messages/apt2265 for details.</value>
```

Comment:
ditto . `https://learn.microsoft.com/dotnet/android/messages/apt2265 ` should work fine